packages/recall-api/src/express.ts

```diff
@@ -82,7 +83,7 @@ export default async ({ app }: { app: Application }) => {
      * Add more versions of the api below
      */
 
-  // app.use('/api/v1');
+  app.use('/api/v1/users', userRoutes);
```

Comment:
The routes are auto-registered if you use the decorators in the `routing-controllers` package. You can read more [here](https://github.com/typestack/routing-controllers?tab=readme-ov-file#example-of-usage)